dts/arm/st/f3/stm32f373.dtsi

```diff
@@ -11,6 +11,11 @@
 	soc {
 		compatible = "st,stm32f373", "st,stm32f3", "simple-bus";
 
+		rcc: rcc@40021000 {
+			/* ADC prescaler is the same as on STM32F1 */
```

Comment:
This comment could be confusing (for someone not aware of the global PR context).

Reply:
Done in #60332